index.html

```diff
@@ -296,4 +297,125 @@ <h2>About me</h2>
       crossorigin="anonymous"
     ></script>
   </body>
+
+  <footer id="contact">
+    <div class="footer-text">
+      <h2 class="title">Contact me</h2>
+      <p class="supporting-text">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+    </div>
+    <form
+      action="https://formspree.io/f/xwkyvenp"
+      method="get"
+      class="form"
+      id="form"
+    >
+      <div class="name">
+        <label for="name"></label>
+        <input
+          type="text"
+          name="fullName"
+          id="name"
+          placeholder="Full name"
+          required
+          minlength="2"
+          maxlength="20"
```

Comment:
[ ] kindly change the name's `max-length to 30` as it's mentioned in the requirements.